OWNERS_ALIASES

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners#owners_aliases
+
+aliases:
+  qe-approvers:
+  - juzhao
+  - lihongyan1
+  doc-approvers:
+  - rkratky
```

Comment:
we need @bburt-rh instead

Reply:
@simonpasquier , looks like @bburt-rh is not yet part of openshift GH org. Should I revert back to @ rkratky?